test/common/unit/test_cryptographer.py

```diff
@@ -101,13 +99,53 @@ def test_decrypt():
     assert Cryptographer.decrypt(encrypted_data, key) == data
 
 
+def test_generate_key():
+    # Not much to test here in the current approach, this simply shadows PrivateKey()
+    assert isinstance(Cryptographer.generate_key(), PrivateKey)
+
+    # Making sure multiple calls to the function do not return the same key
+    issued_keys = []
```

Comment:
Might be better to make `issued_leys` a `set`.

Reply:
Not a big deal given it is only a test.